packages/flame/lib/src/components/mixins/hoverable.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:meta/meta.dart';
+
+import '../../../game.dart';
+import '../../game/base_game.dart';
+import '../../gestures/events.dart';
+import '../base_component.dart';
+
+mixin Hoverable on BaseComponent {
+  bool isHovered = false;
```

Comment:
Nice! We don't have anything like this on the other mixins, I think this would be pretty good to have on `Draggable` too.